### ISSUE TITLE
No planning on support IE 11 ?

### ISSUE THREAD

**Issue by SightStudio** [json-editor/json-editor#776]
### General information

- json-editor version:

im using **json-editor version 2.2.0**

### Expected behavior

i tested feature below
https://github.com/json-editor/json-editor#support-legacy-browser-not-tested

with

```
webpack 3.x with babel and @babel/pollyfill
vue 2.x
```

on **IE 11**

### Actual behavior
Not working on IE.
it throws Syntax Error

### Steps to reproduce the behavior
Since the library contains arrow function which IE dosen't support,
IE throws error while evaling code from chunk

![image](https://user-images.githubusercontent.com/50672087/85350441-1f89a100-b53c-11ea-9e37-4f00da9f0795.png)  
( the word "구문 오류" stands for syntax error in korean )

![image](https://user-images.githubusercontent.com/50672087/85350566-6aa3b400-b53c-11ea-9434-b008a3e31ab3.png)

i know this library is outstanding and it could be hard work.. 
but i'm wondering is there a planning on support at least IE  11 ? via apply babel or others...

**Comment by schmunk42** [comment]
If you can create a PR we'll check it, but we won't be able to do the support.

**Comment by SightStudio** [comment]
ok i'll try

**Comment by devopsix** [comment]
@SightStudio I am also considering using json-editor for an app which has to support IE11.

Below steps got me surprisingly far:

1. Transpile jsoneditor.js using webpack/Babel
1. Include `core-js/es` polyfills
1. Provide an extra polyfill for [`DOMTokenList.add()` (multiple arguments)](https://developer.mozilla.org/en-US/docs/Web/API/DOMTokenList/add) (for a quick poc I used [this one](https://stackoverflow.com/a/23316665/12999078))

I haven't done thorough testing yet. But json-editor seems to properly render and edit a medium complexity object graph (using bootstrap4 theme and fontawesome5 iconlib) in IE11/W10.

For the transpilation I added these dev dependencies:
`npm i --save-dev babel-loader @babel/core @babel/preset-env`

And I added this rule to the webpack configuration:
```typescript
{
  test: /[\/\\]@json-editor[\/\\]json-editor[\/\\]dist[\/\\].+\.js$/,
  use: {
    loader: 'babel-loader',
      options: {
        presets: ['@babel/preset-env']
      }
    },
  }
}
```
